main.nf

```diff
@@ -211,6 +211,8 @@ workflow {
         AnnotateKmers(sig_kmer, reftxt, gff_files)
         genehit = AnnotateKmers.out.annotated_kmers_out
 
-        GeneHitPlot(genehit)
+        plot_script = Channel.value(file("${projectDir}/scripts/gene_hit_summary_plot.R"))
```

Comment:
I find it a bit odd that this is necessary - it feels wrong to supply the script as a value channel to the process. Can't we workaround this using a more native mechanism, e.g. putting it inside a `bin` directory, instead of `scripts` in the root of this repo. I think nextflow will automatically make such scripts available even running within a singularity container.